packages/e2e-tests/tests/highlighter.test.ts

```diff
@@ -0,0 +1,30 @@
+import test, { expect } from "@playwright/test";
+
+import { openDevToolsTab, startTest, selectNodePicker } from "../helpers";
+import { openConsolePanel, warpToMessage } from "../helpers/console-panel";
+
+test("element highlighter works everywhere", async ({ page }) => {
+  await startTest(page, "doc_inspector_basic.html");
+
+  await openDevToolsTab(page);
+  await openConsolePanel(page);
+  await warpToMessage(page, "ExampleFinished");
+
+  await selectNodePicker(page);
+
+  await page.waitForSelector(`[data-value="myiframe"]`);
+
+  const markupNode = await page.locator(`[data-value="myiframe"]`);
+
+  await markupNode.click();
+
+  const highlighter = page.locator("#box-model-content");
+
+  await highlighter.waitFor();
+
+  const pathDefinition = await highlighter.getAttribute("d");
+
+  const pathDefinitionToCompare = `M10,48.400001525878906 L310,48.400001525878906 L310,198.39999389648438 L10,198.39999389648438`;
```

Comment:
I wonder if there's a way we could avoid hard-coding SVG coordinates in this test?

We don't have to figure that out now, since it was already like this. Just thinking out loud.

Reply:
Yeah, Holger has a better approach here that he used for the react inspector we should be able to copy here in a follow-up. Left a TODO comment with a ticket attached 😎